src/public/modules/forms/admin/controllers/edit-fields-modal.client.controller.js

```diff
@@ -9,6 +9,7 @@ const {
   MAX_UPLOAD_FILE_SIZE,
 } = require('shared/constants')
 const { UPDATE_FORM_TYPES } = require('../constants/update-form-types')
+const { bytesToMegabytes } = require('../../../../../app/constants/filesize')
```

Comment:
since this is the client, I don't think it can access `/app`, which may be why the CI/CD is failing. May want to move the constant to the `/shared` folder.